contributions/catalog/gunicorn.config.py

```diff
@@ -0,0 +1,8 @@
+"""Gunicorn configuration."""
+
+bind = '0.0.0.0:5000'
```

Comment:
You are using 5050 in your rest service and 5000 in here. Could you double check if this is okay?